specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Since we won't get a list of ballot marks, I'd drop that, and at some point change `ballot` to an `acvr` defined just like a cvr, (or perhaps that can just be a second `cvr` here?). Finally add one `modification_timestamp` (do you just say `string` here? or `modification_timestamp: string`?), defined as the last modification time.

Note that audit board input like "ballot doesn't exist" need to be encoded as an acvr with votes for all the losers (a `phantom_ballot`). I'm not sure how/where that is specified or implemented (client or server?). Ahh, perhaps that is why kiniry was specifying `ballot` here in the first place, along with the list of marks.